### ISSUE TITLE
My training loss looks very good, and the verification loss has been increasing from the beginning.

### ISSUE THREAD

**Issue by rookiexiao123** [Kohulan/DECIMER-Image-Segmentation#22]
Can you help me see what the problem is？
![微信图片_20210730132949](https://user-images.githubusercontent.com/32377884/127605318-1b065907-7d75-4fa4-af56-d66a77cc1e36.png)

**Comment by rookiexiao123** [comment]
I tried to reduce the learning rate to 0.0001, which is normal. I also want to split the chemical formula, but my annotation is not very fine. Is the author's annotation as fine as that of the mask RCNN balloon?

**Comment by OBrink** [comment]
How big are your training and validation datasets? I do not think that the learning rate is the problem here. This looks more like a case of overfitting. The model performs well on the training data but performs poorly on the validation data. My guess is that the training dataset is too small.
We used a high-quality dataset of manually annotated pages. I can't really make a clear statement without having more information about your data.

**Comment by OBrink** [comment]
@rookiexiao123 Has the issue been resolved?

**Comment by rookiexiao123** [comment]
I think it is ok. I set the step is 1000,but my dataset size is 160.I think the step set too large.now, I set the step = dataset / batchsize. Loss looks much more normal.

**Comment by OBrink** [comment]
@rookiexiao123 Thank you for sharing!

**Comment by rookiexiao123** [comment]
@OBrink  think you very much!
